src/lib/pages/instantiate/instantiate.tsx

```diff
@@ -266,7 +287,7 @@ const Instantiate = ({ onComplete }: InstantiatePageProps) => {
   );
 
   return (
-    <>
```

Comment:
react fragment looks good to me. Is there any problem with it?

Reply:
This currently works fine